### ISSUE TITLE
Purchase; Unable to switch to 5 year plan. 

### ISSUE THREAD

**Issue by Nephilimi** [standardnotes/forum#3370]
I received an email about the black Fridays sale and it sounds good. In order checkout It presents me with the two levels and I'm looking at checkout for "Subscribe to Standard Notes Productivity for $45.00/year".

At the bottom of the page is "change plan" with takes me to a page where I can select the two plan levels. On the bottom of that page is "change to 5 year pricing" Clicking the five year pricing takes me right back to "Subscribe to Standard Notes Productivity for $45.00/year". Only for the briefest instant can I see the 5 year plans and their prices before the page switches back to the 1 year plan. 

So far I've reproduced this in iphone, ipad, and windows/vivaldi. 

https://github.com/standardnotes/forum/assets/26043969/0d7a9df1-68ca-4a4b-8e06-646e32c910ed

**Comment by effieeee** [comment]
@Nephilimi So far have you also been unable to select the 5 year plan when you try to purchase after signing into your account on either the desktop app or the web app? (app.standardnotes.com)

After signing in there, please go to **Preferences > Account** to find the option to subscribe:
![image](https://github.com/standardnotes/forum/assets/54989748/6f5c2529-1c22-484b-ade1-a2206b0a3377)

**Comment by Nephilimi** [comment]
OK, to clarify my first attempt had me logging in via the web page, that was looping me into the 1 year plan as I show in that video. I got to that page by clicking the link in the black friday email which is simply https://standardnotes.com/plans. It had me go through the usual login / 2FA etc, but just couldn't get to the 5 year plan. ¯\_(ツ)_/¯

But on this windows computer I have the app installed and I clicked the button you picture in the windows app, that points me to my same browser that I was having trouble with but this time I can select the five year plan. I'm not clear what the difference was and I was already logged in on this browser when the app bounced me to the payment session, but it worked that time. 

![standardDone](https://github.com/standardnotes/forum/assets/26043969/d986f317-672d-4c73-80ef-a7c5f84fe380)
